docs/core/compatibility/windows-forms/6.0/optimize-scalecontrol-calls.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Breaking change: Virtual method call optimized"
+description: Learn about the breaking change in .NET 6 for Windows Forms where ScaleControl is only called if scaling is needed.
+ms.date: 02/08/2022
+---
+# ScaleControl called only when needed
+
+Scaling is usually needed only when an application is running in <xref:System.Windows.Forms.HighDpiMode.SystemAware> or <xref:System.Windows.Forms.HighDpiMode.PerMonitorV2> mode and the monitor has custom DPI settings that differ from the machine where the app was designed. In these scenarios, the Windows Forms runtime calculates the scale factor, based on custom DPI settings of the monitor, and calls <xref:System.Windows.Forms.Form.ScaleControl(System.Drawing.SizeF,System.Windows.Forms.BoundsSpecified)> with the new scale factor. To improve performance, `ScaleControl` is now called only when the calculated scale factor is something other than 1.0F (that is, scaling is needed). This change can break your app if it overrides `ScaleControl` and performs any custom actions in the override.
+
+## Version introduced
+
+.NET 6 servicing
```

Comment:
I do not think it matters here given that target framework of application does not differentiate between these two. I am not sure how other PU changes are following here.

Reply:
@dreddy-work I think it would be good to put the exact version in. I see it was at least in by 6.0.101 - https://github.com/dotnet/winforms/issues/6613.

Reply:
Agree. I just wasn't clear if other PUs were also following that or simply saying servicing. given they act like inplace at runtime instead of side by side.